pkgs/applications/editors/kakoune/plugins/deprecated.json

```diff
@@ -1 +1 @@
-{}
+{}
```

Comment:
Please add a final new line back.

Reply:
Added again, thanks.
Also fixed the script to add the newline automatically.